include/cantera/oneD/Domain1D.h

```diff
@@ -511,9 +511,20 @@ class Domain1D
         return m_z;
     }
 
+    //! Set up initial grid.
+    //! @since New in %Cantera 3.2.
+    void setupGrid(const vector<double>& grid);
```

Comment:
Can we avoid introducing this overload for `setupGrid`? If we did this for every method that took `double*`, we'd have a _lot_ of extra boilerplate code, and I don't want to force users to use `std::vector`.

Reply:
This is a little more than meets the eye. The current `setupGrid(size_t n, const double* z)` is an outlier as it takes the size as an additional argument rather than just the pointer. We have the alternative of introducing a `setupGrid(const double* z)` and deprecate the current version to be consistent with the remainder of our API. I opted for the inherently safer version of a `std::vector`.